README.md

```diff
@@ -27,6 +27,7 @@ See also: [Getting started with the Görli Testnet](https://mudit.blog/getting-s
   - https://goerli.ethstats.io/
 - Block Explorer: 
   - https://goerli.etherscan.io/
+  - https://goerli.ethstats.io/
```

Comment:
See line 27, do you want to have two entries here or shall we move it down?

Reply:
All the entries. No, I missed the fact that it was in the wrong category. We also have a status dashboard / network monitor but that will be ready soon. I removed the superfluous entry. Thanks!